extensions/ql-vscode/gulpfile.ts/package.ts

```diff
@@ -17,6 +17,7 @@ export async function packageExtension(): Promise<void> {
       "..",
       `${deployedPackage.name}-${deployedPackage.version}.vsix`,
     ),
+    "--no-dependencies",
```

Comment:
The docs are a little unclear as to what this option does. Does this mean that `node_modules` will be ignored when packaging the extension? And are we able to do this because we are now using esbuild, which does the bundling for us?

Reply:
Thanks for the explanation.